resource/doc/libvmaf.md

```diff
@@ -54,9 +54,58 @@ Using FFmpeg with libvmaf is very powerful, as you can create complex filters to
+aligned. We can also the `-an` option to disregard the presence of audio.
 
-See the [FFmpeg's guide to libvmaf](https://ffmpeg.org/ffmpeg-filters.html#libvmaf), the [FFmpeg Filtering Guide](https://trac.ffmpeg.org/wiki/FilteringGuide) for more examples of complex filters, and the [Scaling Guide](https://trac.ffmpeg.org/wiki/Scaling) for information about scaling and using different scaling algorithms.
+See the [FFmpeg's guide to libvmaf](https://ffmpeg.org/ffmpeg-filters.html#libvmaf), the [FFmpeg Filtering Guide](https://trac.ffmpeg.org/wiki/FilteringGuide) for more examples of complex filters, and the [Scaling Guide](https://trac.ffmpeg.org/wiki/Scaling) for information about scaling and using different scaling algorithms. Please refer to the next section for some additional examples on more complex cases.
+
+### Use `libvmaf.a` with FFmpeg - More examples
+
+The above command line works well for most cases, but additional steps might be necessary.
+
+For example, when the frame rates between reference and encoded video do not match, e.g., when the source is 60 fps and the encode is 30 fps:
+
+```
+ffmpeg -r 1 -i main.mpg -r 2 -i ref.mpg -filter_complex \
+"[0:v]scale=1920x1080:flags=bicubic,setpts=PTS-STARTPTS[main];[1:v]select='not(mod(n,2))',setpts=PTS-STARTPTS[ref];[main][ref]libvmaf" -an -f null -
+```
+
+Here we explicitly state that the reference video has double the frame rate and also use the select filter to select half of the reference frames to calculate VMAF.
+
+If we want a specific model to be used (e.g. the `baz.pkl` model file under the `/foo/bar/` directory):
+
+```
+ffmpeg -i main.mpg -i ref.mpg -filter_complex \
+"[0:v]scale=1920x1080:flags=bicubic,setpts=PTS-STARTPTS[main];[1:v]setpts=PTS-STARTPTS[ref];[main][ref]libvmaf='model_path=/foo/bar/baz.pkl'" -an -f null -
+```
+
+If we want to specify a model that relies on bootstrapping (e.g. the 0.6.3 model file under the `/foo/bar/` directory), the `enable_confidence_interval` option needs to be set:
+
+```
+ffmpeg -i main.mpg -i ref.mpg -filter_complex \
+"[0:v]scale=1920x1080:flags=bicubic,setpts=PTS-STARTPTS[main];[1:v]setpts=PTS-STARTPTS[ref];[main][ref]libvmaf='model_path=/foo/bar/vmaf_b_v0.6.3.pkl:enable_conf_interval=1'" -an -f null -
+```
+
+If we want to store the results in an XML file with a given name (e.g. under `/foo/bar/baz/results.xml`):
+
+```
+ffmpeg -i main.mpg -i ref.mpg -filter_complex \
+"[0:v]scale=1920x1080:flags=bicubic,setpts=PTS-STARTPTS[main];[1:v]setpts=PTS-STARTPTS[ref];[main][ref]libvmaf='log_path=/foo/bar/baz/results.xml:model_path=/foo/bar/baz.pkl'" -an -f null -
+```
+
+If we want to enable the calculation of an additional metric, like SSIM:
+
+```
+ffmpeg -i main.mpg -i ref.mpg -filter_complex \
+"[0:v]scale=1920x1080:flags=bicubic,setpts=PTS-STARTPTS[main];[1:v]setpts=PTS-STARTPTS[ref];[main][ref]libvmaf='ssim=1:model_path=/foo/bar/baz.pkl'" -an -f null -
+```
+
+If we want use a single thread:
```

Comment:
Would it make sense to explain why someone would want that? Is multithreaded computation non-deterministic, like with some video encoders?

Reply:
The purpose of this example is to just showcase how the user can control the number of threads. One use-case is to compare the execution time of a single-threaded vs. a multi-threaded execution.

Reply:
I can add a comment in the docs of the above use case.